skbio/stats/tests/test_composition.py

```diff
@@ -269,6 +335,92 @@ def test_centralize(self):
                             np.array([[2, 2, 6],
                                       [4, 4, 2]]))
 
+    def test_ancom_basic(self):
```

Comment:
Also would be great to test that the inputs (`table`, `grouping`) are not modified in-place by the method.